deploy/crds/keycloak.org_keycloakrealms_crd.yaml

```diff
@@ -265,6 +265,11 @@ spec:
                             type: string
                           description: Client Attributes.
                           type: object
+                        authenticationFlowBindingOverrides:
+                          additionalProperties:
+                            type: string
+                          description: Authentication Flow Binding Overrides.
+                          type: object
```

Comment:
This does not belong to the Realm CRD.

Please do not make manual changes to the CRDs and always run `make code/gen`. Thank you.

Reply:
uhm i just reverted the yaml changes and ran the command and the result is identical to my changes…?

Reply:
@vmuzikar can you verify this?

Reply:
@vmuzikar can you verify that?

Reply:
Sorry again for the late response. I realized it is actually ok. The Client API representation is really included in the Realm CR as it is possible to define Clients there. Sorry about the confusion.

Reply:
no worries, i wasnt following the proper steps in my first try so it couldve just as well been an error on my side so i verrified, so even though unintentionally, you made me follow proper procedures :P